lib/modules/home/components/slide_item.dart

```diff
@@ -0,0 +1,89 @@
+part of '../home_module.dart';
+
+class SlideItem extends StatelessWidget {
+  final SurveyInfo survey;
+  const SlideItem({
+    Key? key,
+    required this.survey,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      fit: StackFit.expand,
+      children: [
+        Bone.hidden(
+          child: Image(
+            image: NetworkImage(survey.coverImageUrl!),
+            fit: BoxFit.fill,
+          ),
+        ),
+        Align(
+          alignment: Alignment.bottomCenter,
+          child: SafeArea(
+            child: Container(
+              margin: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+              child: SizedBox(
+                height: 126,
+                child: Row(
+                  children: [
+                    Expanded(
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          Bone.multiple(
+                            number: 2,
+                            variants: const [1, 0.5],
+                            child: Text(
+                              survey.title!,
+                              style: const TextStyle(
+                                color: Colors.white,
+                                fontSize: 28,
+                              ),
```

Comment:
currently the layout is break if we use long text, could you please recheck and handle it? :sweat: